src/state/sagas.js

```diff
@@ -124,7 +138,22 @@ export async function fetchAnnotationResource(url) {
 
 /** Saga for fetching external annotation resources */
 export function* fetchExternalAnnotationResources({ targetId, annotationId, annotationJson }) {
-  if (!annotationJson.resources.some(hasExternalResource)) {
+  if (naiveIIIFv3Check(annotationJson)) {
+    // We treat this as IIIF 3.0
+    yield fetchExternalAnnotationResourceIIIFv3({ targetId, annotationId, annotationJson });
```

Comment:
Did you verify that this works? I think delegating to other sagas should either use the `call` effect or `yield from` to yield all of the effects from the delegated saga?